src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-card flat id="upload-memorandum">
+
+    <!-- Summary Section -->
+    <div id="upload-memorandum-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="upload-memorandum-summary-header" >
+        <v-icon color="#38598A">mdi-text-box-multiple</v-icon>
+        <label class="upload-memorandum-title"><strong>Memorandum</strong></label>
+      </div>
+
+      <!-- Summary error message -->
+      <div v-if="!valid" class="upload-memorandum-error-message">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text"> This step is not complete. </span>
+          <router-link :to="{ path: `/${RouteNames.CREATE_MEMORANDUM}`, query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-card-title>Memorandum Placeholder component</v-card-title>
```

Comment:
Title at the end? Or title element used for emphasis?

Reply:
Yea, no real significance here :) Just slapped some content in there for display purposes. There are just placeholders, 100% subject to change.

Reply:
OK, next person to work in these files can fix it.